libs/utils/aws/src/lib/image.ts

```diff
@@ -81,6 +81,29 @@ export class Image {
     return newImage;
   }
 
+  static _getNewSameRatioSize = (
+    originalWidth: number,
+    originalHeight: number,
+    size: Size
+  ) => {
```

Comment:
```suggestion
  ): Size => {
```

I think we should annotate with type instead of using `as`.